studio.py

```diff
@@ -61,6 +61,19 @@ def _digest_file(self, file, algorithm):
             print("hashed %s with %s to be %s" % (name, algorithm, h.hexdigest()))
             return h.hexdigest()
 
+    def replay_site(self, site, screen_shot=False):
```

Comment:
This appears to mostly duplicate `replaying`. Can we reduce the duplication? If replaying currently waits for the user to kill the process, maybe we should change it to the 5 second sleep and screenshot by default instead.

Reply:
Updated to run the same method